libevm/VMFactory.cpp

```diff
@@ -28,15 +31,20 @@
 #include <hera.h>
 #endif
 
+namespace dll = boost::dll;
+namespace fs = boost::filesystem;
 namespace po = boost::program_options;
 
+using evmc_create_fn = evmc_instance*();
+
 namespace dev
 {
 namespace eth
 {
 namespace
 {
 auto g_kind = VMKind::Legacy;
```

Comment:
logically these two globals should be private static members of `VMFactory`

Reply:
True, but then I will have to expose them in the header file. I prefer not to. However, not sure if the prefix should be `s_` or `g_` :)

Reply:
here `g_`; `s_` is for local statics